integration_testing/features/super-admin/super-admin-change-peer-unlisted-import.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Super admin changes device setting allowing unlisted channel peer import
+  Super admin needs to be able to change the setting to allow other network computers to import the device's unlisted channels
+
```

Comment:
Could you add a comment here for the testers that for this scenario they must have another Kolibri running on a discoverable device in the same LAN?